zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -119,13 +119,12 @@ else
     bashio::log.info "Soft-reset disabled by user"
 fi
 
-presets=""
+safe_mode=""
```

Comment:
I'd still use only one value for all preset values as we would need to build a formatted JSON list of all presets before updating the template.